### ISSUE TITLE
Configurable tolerance

### ISSUE THREAD

**Issue by amannn** [gabrielbull/react-aim#5]
Thanks a lot for this library! I really appreciate how flexible it can be used for different use cases.

I was wondering if it could be possible to allow the consumer of the component to configure the level of tolerance?

E.g. in your [menu example](http://gabrielbull.github.io/react-aim/): When hovering over the menu, the items change their state very responsively when moving the mouse straight up and down or of course slightly to the left. However, if the mouse is moved slightly to the right the menu isn't so responsive anymore. Of course, there needs to be a boundary somewhere, but it would be nice if this was configurable.

Thanks in advance!

**Comment by gabrielbull** [comment]
@amannn I agree that it would be nice if it could be configured. If you have time, feel free to submit a PR.

**Comment by cwick** [comment]
We need this too for our project, so we may look into adding some kind of minimum velocity setting.

**Comment by gabrielbull** [comment]
@cwick please feel free to submit a PR.
